Bug fixes from review (#3)
node-red/node-red-dev-cli: docs/P02.md

# Check package name in repository is the same name as the package.json

## Requirement

The name field in the package.json SHOULD match the name field in the package.json hosted within the respository specified in the package.json.

## Reason

> This is to check for forked nodes where the publisher has forked an original repo and publihsed the node under a new name but not updated the repo to point to their own new copy. This can result in authors of the original node getting issues raised against their repoisitory for a downstream fork.

Reviewer: ```suggestion
This is to check for forked nodes where the publisher has forked an original repo and published the node under a new name but not updated the repo to point to their own new copy. This can result in authors of the original node getting issues raised against their repository for a downstream fork.
```

## Note 

> This test may fail where you are validating a local copy of the package and it has not yet been published, or where the code has not been published in a repoository.

Reviewer: ```suggestion
This test may fail where you are validating a local copy of the package and it has not yet been published, or where the code has not been published in a repository.
```

Where code is part of a mono-repo you should use the `directory` key within `repository` to point to the path where the package.json is location

## Reference

> https://docs.npmjs.com/cli/v7/configuring-npm/package-json#name

Reviewer: ```suggestion
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository
```
